src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -217,26 +226,64 @@ std::unique_ptr<data_iterator> drawing_classifier::create_iterator(
 
   data_params.is_train = is_train;
   data_params.target_column_name = read_state<flex_string>("target");
-  data_params.feature_column_name = read_state<flex_string>("feature");
+  const flex_list &features_values = read_state<flex_list>("features");
+  DASSERT_GE(features_values.size(), 1);
+  data_params.feature_column_name = features_values[0].get<flex_string>();
```

Comment:
since the feature_list only contains one element, why bother using a list?

Reply:
I think there's an argument to be made both for and against using a feature list but from https://github.com/apple/turicreate/pull/2439/files#r334137025, I thought we agreed that we'll stick with a single-element feature list. This would also give us the flexibility to add more features down the road if we want to. I do agree that it is a little awkward since the data_params only has one feature_column_name as a string and not a vector.

Reply:
There's one philosophy called [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)

Reply:
Haha okay I was always convinced to have just one feature like I mentioned in the comment thread on #2439. I'll make the software change to take care of that in this PR.

Reply:
Sorry, I take that back. Changing to a single feature name would require a reasonable code change in init_model and export_coreml functionality and tests. I think we should table that change for convergence, and not as part of this PR.